docs/vNext/Fabric_Configuration.md

```diff
@@ -35,13 +35,13 @@ The adapter exposes many SDK features directly to the user callback modules, mak
 
 You must bind Caliper to a specific Fabric SDK to target the corresponding (or compatible) SUT version. Refer to the [binding documentation](./Installing_Caliper.md#the-bind-command) for details.
 
-### Binding with Fabric SDK 2.2.x
-> Note that when using the binding target for the Fabric SDK 2.2.x there are capability restrictions:
-> * The 2.2 SDK does not facilitate administration actions. It it not possible to create/join channels, nor install/instantiate contract. Consequently the 2.2 binding only facilitates operation with a `--caliper-flow-only-test` flag
-> * The 2.2 SDK currently only supports operation using a `gateway`. Consequently the 2.2.x binding requires a `--caliper-fabric-gateway-enabled` flag
+### Binding with Fabric SDK 2.1.x
```

Comment:
Maybe we should use 2.x here until (when/if) the admin capabilities are put back? It's easier to maintain, and still correct currently.